plugin/grpctrace/interceptor.go

```diff
@@ -43,9 +43,29 @@ var (
 	messageUncompressedSizeKey = kv.Key("message.uncompressed_size")
```

Comment:
Should this be `messaging.message_payload_size_bytes` according to https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/messaging.md or these are unrelated?